### DISCUSSION TITLE
How to use variable fonts?

### DISCUSSION THREAD

**Discussion by srid** [tw-in-js/twind#169] [General]
It'd be great if someone could provide an example of how to use variable fonts when using twind.

[tailwind's own site](https://github.com/tailwindlabs/tailwindcss/discussions/3225) uses the `Inter` font which is available on Google Fonts, but the `font-extrabold` thing doesn't bold it in the say way the font is bolded on tailwind's site.

See also https://github.com/tailwindlabs/tailwindcss/discussions/3225

**Comment by sastan** [comment]
That may get solved with the capsize plugin @lukejacksonn  is working on.

**Comment by lukejacksonn** [comment]
It says in that post that you need `font-variation-settings: 'wght' 700;` to style variable fonts. You could achieve this in a few ways. Making your own plugin is probably the easiest. Maybe something like:

```js
plugins: {
  vfont: (parts, { theme }) => ({
    'font-variation-settings': `'wght' ${theme('fontWeight', parts[0])}`;
  })
}
```

This would allow you to do `vfont-extrabold` and the appropriate `font-variation-settings` weight would be applied.

Let me know if this helps!
